README.md

```diff
@@ -1,2 +1,9 @@
-# test1
-collaboration exploration
+# Test Repository 
+
+This is a test repository to practice proper GIT methodology between an owner and a contributor who will fork the project.
```

Comment:
This was added by clicking on line 3. In theory, I'd comment on the text of that paragraph.